pysat/utils/_core.py

```diff
@@ -702,9 +702,32 @@ def display_available_instruments(inst_loc=None, show_inst_mod=None,
 class NetworkLock(Lock):
     def __init__(self, *args, **kwargs):
         """Lock manager compatible with networked file systems
+
+        Parameters
+        ----------
+        *args : list reference
+            References a list of input arguments
+        **kwargs : dict reference
+            References a dict of input keyword argument
```

Comment:
Since the allowed inputs aren't explained here, this docstring should state which docstring a user should go to for a detailed description.

Reply:
Should I point to the portalocker class this object inherits from or  go ahead and include information from that docstring here?

Reply:
Just point to the portalocker class.  That way, the docs remain good if the inputs change.

Reply:
I have added a sphinx extension intersphinx to make referring to the portalocker docs (and any others down the line) "simple". I can easily remove this and just add a link.

Reply:
I was thinking of something simpler and more ipython friendly like adding 'see portalocker.NetworkLock'.

Reply:
That's a good point. I had not looked at the output in ipython. It actually pulls the documentation down from the inherited class. Also in the current state the ipython output has :class:`portalocker.utils.Lock` written out. I think in ipython the difference is small, but in readthedocs this is slightly more convenient. But it adds to the complexity of the sphinx conf.py. So this does seem ipython friendly, but I'm still happy to make this simpler.

Reply:
Why don't we keep what you did but also add "See portalocker.NetworkLock for more details" as a Note.

Reply:
Sounds good. Updated for this discussion